sklearn/compose/tests/test_column_transformer.py

```diff
@@ -1180,3 +1180,15 @@ def test_column_transformer_mask_indexing(array_type):
     )
     X_trans = column_transformer.fit_transform(X)
     assert X_trans.shape == (3, 2)
+
+
+def test_n_features_in():
+    # make sure n_features_in is what is passed as input to the column
+    # transformer.
+
+    X = [[1, 2], [3, 4], [5, 6]]
+    ct = ColumnTransformer([('a', DoubleTrans(), [0]),
+                            ('b', DoubleTrans(), [1])])
+    assert not hasattr(ct, 'n_features_in_')
```

Comment:
Would it be an issue if it already has that here? What exactly are we testing?

Reply:
we're testing that the `n_features_in_` attribute doesn't exist unless `fit` has been called before

Reply:
yeah but why?